src/wizard/apple/ios.md

```diff
@@ -5,11 +5,9 @@ support_level: production
 type: language
 ---
 
-We support installing the SDK with [CocoaPods](/platforms/apple/install/cocoapods/), [Swift Package Manager](/platforms/apple/install/swift-package-manager/), and [Carthage](/platforms/apple/install/carthage/).
+## Install
 
-### CocoaPods
-
-To integrate Sentry into your Xcode project, specify it in your `Podfile`:
+We recommend installing the SDK with CocoaPods, but we also support [alternate installation methods](/platforms/apple/install/). To integrate Sentry into your Xcode project, specify it in your Podfile:
```

Comment:
Since the goal is to increase conversion rate, I want to suggest to change from cocoapods to SPM as recommendation. 
While cocoapods requires devs to have it, SPM works out of the box with Xcode, IMO much more easy to use and friendly for new comers.